llama-index-core/tests/token_predictor/test_base.py

```diff
@@ -26,23 +26,18 @@ def test_token_predictor(mock_split: Any) -> None:
     )
     document = Document(text=doc_text)
     llm = MockLLM(max_tokens=256)
-    service_context = ServiceContext.from_defaults(llm=llm)
 
     # test tree index
-    index = TreeIndex.from_documents([document], service_context=service_context)
+    index = TreeIndex.from_documents([document])
```

Comment:
we should have `index = TreeIndex.from_documents([document], llm=llm)` here (the tree index uses the LLM to build)

Reply:
Not sure if it matters since this is tests I guess

Reply:
Seems like tests pass, so I trust